.github/workflows/packaging.yml

```diff
@@ -148,16 +148,19 @@ jobs:
       - name: Freeze application
         run: python3 build-aux/windows/setup.py bdist_msi
 
+      - name: Make package portable
+        run: touch build-aux/windows/build/package/Nicotine+/portable
```

Comment:
`touch .../portable` -> `mkdir .../userdata`

The presence of a folder should be enough and seems less obfuscated than needing the blank file as well.

Also consider "appdata" instead of "userdata".